<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewTrace</class>
 <widget class="QDialog" name="NewTrace">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>398</width>
    <height>549</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Create a new derivation tree</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="5" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <layout class="QFormLayout" name="formLayout">
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::ExpandingFieldsGrow</enum>
     </property>
     <property name="horizontalSpacing">
      <number>0</number>
     </property>
     <property name="verticalSpacing">
      <number>0</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="labelnewtrace">
       <property name="text">
        <string>Name:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="graphname">
       <property name="text">
        <string>New derivation tree</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="4" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Derivation Tree Type</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QRadioButton" name="StandardTree">
        <property name="text">
         <string>Standard Derivation &amp;Tree</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="SmAnim">
        <property name="text">
         <string>Small Multiples/Animation </string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="topMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QRadioButton" name="HistogramTree">
          <property name="text">
           <string>Scatter plot: count nu&amp;mber of</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListWidget" name="portSelectionlistWidget">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <property name="alternatingRowColors">
           <bool>true</bool>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::MultiSelection</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayoutScatterplotProperty">
        <item>
         <widget class="QRadioButton" name="PropertyHistogramTree">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>Scatter plot:</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayoutTypeElement">
          <item>
           <widget class="QLabel" name="labelelement">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="text">
             <string>count the number of</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="typeElementComboBox">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <item>
             <property name="text">
              <string extracomment="Only consider the information carried by the central node.">Port nodes</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string extracomment="Consider the edges linking port nodes together.">Edges</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string extracomment="Consider all the elements mentionned above.">All elements</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="labelpropertyexplanation">
          <property name="text">
           <string>with a property value verifying the following</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayoutProperty">
          <item>
           <widget class="QComboBox" name="propertySelectionComboBox">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="minimumSize">
             <size>
              <width>140</width>
              <height>0</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="propertyRelationComboBox">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="minimumSize">
             <size>
              <width>85</width>
              <height>0</height>
             </size>
            </property>
            <item>
             <property name="text">
              <string extracomment="is equal to">=</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string extracomment="is inferior to">&lt;</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string extracomment="is inferior or equal to">&lt;=</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string extracomment="is superior to">&gt;</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string extracomment="is superior or equal to">&gt;=</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string extracomment="is different of">!=</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="propertyValueLineEdit">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>0</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayoutRatioProperty">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayoutCurrent">
          <item>
           <widget class="QRadioButton" name="ratioPropertyButton">
            <property name="text">
             <string>&amp;Ratio: Property (t)</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="ratioPropertyCurrentComboBox">
            <property name="enabled">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayoutAntecedent">
          <item>
           <widget class="QLabel" name="ratioPropertyAntecedentLabel">
            <property name="text">
             <string>against property (t-1)</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="ratioPropertyAntecedentComboBox">
            <property name="enabled">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayoutGraphAttribute">
        <item>
         <widget class="QRadioButton" name="GraphAttributeButton">
          <property name="text">
           <string>Scatter plot: use &amp;graph attribute</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="graphAttributeNameEdit">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>NewTrace</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>NewTrace</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
